RecoVertex/PrimaryVertexProducer/interface/DAClusterizerInZT_vect.h

```diff
@@ -0,0 +1,318 @@
+#ifndef DAClusterizerInZT_vect_h
+#define DAClusterizerInZT_vect_h
+
+/**\class DAClusterizerInZT_vect
+
+ Description: separates event tracks into clusters along the beam line
+
+	Version which auto-vectorizes with gcc 4.6 or newer
+
+ */
+
+#include "RecoVertex/PrimaryVertexProducer/interface/TrackClusterizerInZ.h"
+#include "TrackingTools/TransientTrack/interface/TransientTrack.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include <vector>
+#include "DataFormats/Math/interface/Error.h"
+#include "RecoVertex/VertexTools/interface/VertexDistanceXY.h"
+#include "RecoVertex/VertexPrimitives/interface/TransientVertex.h"
+
+#include <memory>
+#include <stdlib.h>
+#include <malloc.h>
+
+template <typename T, std::size_t N = 16>
+class AlignmentAllocator {
```

Comment:
OK - on 1 there is no difference using the aligned allocator vs. the standard one, even for 64 byte alignment on the machine that I'm testing on, which is interesting.

I do see performance differences when I don't require alignment on the __restrict__ pointers, which I find a little weird.